plugins/openstack/src/main/java/org/eclipse/xpanse/orchestrator/plugin/openstack/monitor/keystone/KeystoneManager.java

```diff
@@ -93,6 +93,7 @@ private static String getIpAddressFromUrl(String url) {
     private Config buildClientConfig(String url) {
         return Config.newConfig()
                 .withEndpointNATResolution(getIpAddressFromUrl(url))
-                .withEndpointURLResolver(new CustomEndPointResolver());
+                .withEndpointURLResolver(new CustomEndPointResolver())
+                .withProxy(new ProxyHost("http://proxyde.huawei.com", 8080));
```

Comment:
seems we need some configurations for Monitor too.

Reply:
@iskey good catch. I must remove this. This is needed only for me to work from office network.